whitepapers/Tables.md

```diff
@@ -2,4 +2,40 @@
 
 
 
+## Pipeline configuration
 
+### Source
+A Source is a place metadata comes from.
+
+#### Columns
+* `name` -- Short, unique string
+* `long_title` -- full, human-friendly name
```

Comment:
As things stand `long_title` should be unique as well.

Side note: we should come up with a slightly better format for expressing sql rows. 
Each should convey, if it's unique, indexed, unique with another value, datatype, nullable, default, etc.

Should we just make a table template that we copy for everything?